.circleci/config.yml

```diff
@@ -490,8 +490,8 @@ commands:
               fi
   install_browser_tools:
     steps:
-      - browser-tools/install-chrome
-      - browser-tools/install-chromedriver
+      - browser-tools/install-browser-tools:
```

Comment:
I think this can be reverted since it was fixed in https://github.com/dockstore/dockstore-ui2/pull/1937